packages/block-library/src/columns/index.js

```diff
@@ -79,6 +79,11 @@ export const settings = {
 			},
 		],
 	},
+	__experimentalLabel( _, { context } ) {
```

Comment:
This API sounds very weird to me, using a "block API" to label a block support. Did we consider a config in the block.json file?

Reply:
It is pretty weird, particularly now that I've had a chance to sleep on it! I think a config in `block.json` would be a better way to go about it.

I agree with Joen, though, that it'd be good to try to do the least amount of work here as possible for the Columns block, so in the immediate term, I'm going to revert the label overriding and just set the string to a hard-coded `Block spacing` for the time being.

If and when we get back to it, I'll take a look at seeing if we can set it in the `block.json` file. Thanks for taking a look 🙂